src/middleware/index.js

```diff
@@ -1 +1 @@
-module.exports = (app) => {};
+module.exports = () => {};
```

Comment:
Did you change this because of ESLint? This function accepts app, if implemented.